telemetry/telemetry.go

```diff
@@ -183,11 +186,34 @@ func main() {
 	if *withMasterArbitration {
 		s.ReqFromMaster = gnmi.ReqFromMasterEnabledMA
 	}
-	
-	log.V(1).Infof("Auth Modes: ", userAuth)
+
+	var reload = make(chan int, 1)
```

Comment:
Why do you add this channel? Add some comments?

Reply:
Added comments, reload channel is used to as a way for gnmi server to stop. serve() is now executed in a goroutine and we use reload channel such that if it gets a value, we will stop server and exit.